src/templates/zh-CN/2019/methodology.html

```diff
@@ -392,7 +392,7 @@ <h3 id="analysis"><a href="#analysis" class="anchor-link">分析</a></h3>
       </p>
 
       <p>
-        整个2019年7月, HTTP Archive 数据管道爬取了数百万网站, 收集到的元数据用于了 Web Almanac 网络年鉴.
+        整个2019年7月, HTTP Archive 数据管道爬取了数百万网站, 收集到的元数据用于了 Web Almanac 网络年鉴。
```

Comment:
These the correct commas to use?

```suggestion
        整个2019年7月, HTTP Archive 数据管道爬取了数百万网站、收集到的元数据用于了 Web Almanac 网络年鉴。
```

Reply:
It should be a comma or a full stop.